src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,590 @@
+            proc_ads = self.getProcAds(jobsReady)
+
+            logging.debug("Start: Submitting %d jobs using Condor Python SubmitMany" % len(proc_ads))
+            try:
+
+                clusterId = schedd.submitMany(cluster_ad, proc_ads)
+
+            except Exception as ex:
+
+                logging.error("SimpleCondorPlugin job submission failed.")
+                logging.error("Moving on the the next batch of jobs and/or cycle....")
+                logging.exception(ex)
+
+                condorErrorReport = Report()
+                condorErrorReport.addError("JobSubmit", 61202, "CondorError", str(ex))
+                for job in jobsReady:
+                    job['fwjr'] = condorErrorReport
+                    failedJobs.append(job)
+
+                # Report submission error to operators
+                # TODO - Dirk: Do we really need this ?
+                try:
+                    from WMCore.Alerts import API as alertAPI
+                    preAlert, sender = alertAPI.setUpAlertsMessaging(self,
+                                                                     compName="BossAir.SimpleCondorPlugin")
+                    sendAlert = alertAPI.getSendAlert(sender=sender,
+                                                      preAlert=preAlert)
+                    sendAlert(6, msg="Condor submission failed. Check condor status.")
+                    sender.unregister()
+                except:
+                    pass
+
+            else:
+
+                logging.debug("Finish: Submitting jobs using Condor Python SubmitMany")
+                for index,job in enumerate(jobsReady):
+                    job['gridid'] = "%s.%s" % (clusterId, index)
+                    job['status'] = '1'
+                    successfulJobs.append(job)
+
+        # We must return a list of jobs successfully submitted and a list of jobs failed
+        logging.info("Done submitting jobs for this cycle in SimpleCondorPlugin")
+        return successfulJobs, failedJobs
+
+    def track(self, jobs):
+        """
+        _track_
+
+        Track the jobs while in condor
+        This returns a three-way ntuple
+        First, the total number of jobs still running
+        Second, the jobs that need to be changed
+        Third, the jobs that need to be completed
+        """
+
+        changeList = []
+        completeList = []
+        runningList = []
+
+        # get info about all active and recent jobs
+        logging.debug("SimpleCondorPlugin is going to track %s jobs", len(jobs))
+
+        schedd = htcondor.Schedd()
+
+        logging.debug("Start: Retrieving classAds using Condor Python XQuery")
+        try:
+            itobj = schedd.xquery("WMAgent_AgentName == %s" % classad.quote(self.agent),
+                                  ['ClusterId', 'ProcId', 'JobStatus', 'MATCH_EXP_JOBGLIDEIN_CMSSite'])
+        except:
+            logging.debug("Query to condor schedd failed in SimpleCondorPlugin")
```

Comment:
Don't report errors in `debug` level please. For this case, you could:
`logging.exception("Query to condor schedd failed in SimpleCondorPlugin: ")`